### PR TITLE
fix minor typo on the image segmentation tutorial

### PR DESCRIPTION
**Pull request by jhhan0**
tf.keras.losses.CategoricalCrossentropy should be changed to tf.keras.losses.SparseCategoricalCrossentropy since the pixel classes (labels) are integers, not one-hot encoded. Code is written correct, but the markdown should fix to prevent potential confusion to readers.

### CODE REVIEW

**Comment by jhhan0** [comment]
Everything else is all perfect. Thank you for amazing image segmentation tutorial 👍

**Review by markmcd — APPROVED** [review]
LG, thanks!
